### PR TITLE
added_breadcrumbs_template

### PR DESCRIPTION
**Pull request by gaurav-iitb**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/79"></a>

### CODE REVIEW

File: pages/[content].tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ import Navigation from "../components/ContentTemplates/NavigationTemplate"
 import Tables from "../components/ContentTemplates/TablesTemplate"
 import Video from "../components/ContentTemplates/VideoTemplate"
 import Pagination from "../components/ContentTemplates/PaginationTemplate"
+import Breadcrumbs from "../components/ContentTemplates/BreadcrumbsTemplate"
```
Comment (EmmaDawsonDev) [inline pages/[content].tsx:27]: Can you import this one in the correct place alphabetically?

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just a small change then this looks ready to merge :)

**Comment by gaurav-iitb** [comment]
I have done changes now. please merge it

**Review by EmmaDawsonDev — APPROVED** [review]
Great! Thanks for your contribution!

**Pull request merged**